share/docker/m2/toolchains.xml

```diff
@@ -16,14 +16,15 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
 -->
-<toolchains>
+<toolchains xmlns="http://maven.apache.org/TOOLCHAINS/1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/TOOLCHAINS/1.1.0 http://maven.apache.org/xsd/toolchains-1.1.0.xsd">
   <toolchain>
     <type>jdk</type>
     <provides>
       <version>11</version>
     </provides>
     <configuration>
-      <jdkHome>/usr/lib/jvm/openjdk-11</jdkHome>
+      <jdkHome>/usr/lib/jvm/java-11-openjdk-arm64</jdkHome>
```

Comment:
Where is this toolchain.xml file being used ?
I'd assume that most of the time it is used on x86_64 rather than aarch64 machine.